### ISSUE TITLE
Add '--since' and '--till' options

### ISSUE THREAD

**Issue by chshersh** [feramhq/Transity#12]
I would like to see `transity balance` for specified period of time. Would be good to have some options so I can configure this interval.

**Comment by ad-si** [comment]
Yeah, that would be a very good feature indeed. But we should probably stick with the `--begin` and `--end` that other plain text accounting tools use.
